frame/sudo/src/mock.rs

```diff
@@ -29,6 +29,7 @@ use sp_runtime::{Perbill, traits::{BlakeTwo256, IdentityLookup}, testing::Header
 use sp_io;
 use crate as sudo;
 use frame_support::traits::Filter;
+use frame_system as system;
```

Comment:
Why do you rename it here?

Reply:
`system` from `use super::*;` is used in this mock mod, and after the renaming in `fram/sudo/src/lib.rs` got deleted, there is no `system` in mock any more.

I just double checked and it seems `system` and `frame_system` are both used here. Will change `system` to `frame_system` to make it consistent.